src/test/java/com/michelin/ns4kafka/integration/TopicIntegrationTest.java

```diff
@@ -601,6 +599,13 @@ void shouldDeleteRecordsOnCompactTopic() {
         assertEquals(HttpStatus.UNPROCESSABLE_ENTITY, exception.getStatus());
     }
 
+    private void forceTopicSynchronization() throws InterruptedException {
+        topicAsyncExecutorList.forEach(TopicAsyncExecutor::run);
+
+        // Wait for topics to be updated in Kafka broker
+        Thread.sleep(2000);
```

Comment:
The sleep time seems arbitrary (but seems enough though). Is there any method to wait until the topics is updated?